code/__DEFINES/dcs/signals.dm

```diff
@@ -1343,6 +1343,12 @@
 /// Called in /obj/structure/moneybot/add_money(). (to_add)
 #define COMSIG_MONEYBOT_ADD_MONEY "moneybot_add_money"
 
+/// Called when somebody passes through a scanner gate and it triggers
+#define COMSIG_SCANGATE_PASS_TRIGGER "scangate_pass_trigger"
```

Comment:
Signals should only be sent from one place.

Make it clear this is only for circuit shells, remove its reference in the normal scanner gates (you don't need to make a new signal since it's unused).